sdks/python/apache_beam/testing/load_tests/load_test.py

```diff
@@ -52,32 +52,24 @@ def setUp(self):
     self.input_options = json.loads(self.pipeline.get_option('input_options'))
     self.project_id = self.pipeline.get_option('project')
 
-    self.publish_to_big_query = self.pipeline.get_option('publish_to_big_query')
     self.metrics_dataset = self.pipeline.get_option('metrics_dataset')
     self.metrics_namespace = self.pipeline.get_option('metrics_table')
 
-    if not self.are_metrics_collected():
-      logging.info('Metrics will not be collected')
-      self.metrics_monitor = None
-    else:
-      self.metrics_monitor = MetricsReader(
-          project_name=self.project_id,
-          bq_table=self.metrics_namespace,
-          bq_dataset=self.metrics_dataset,
-      )
+    self.metrics_monitor = MetricsReader(
```

Comment:
I am not familiar with MetricsReader

Reply:
We use MetricsReader to push metrics obtained from PipelineResult to BigQuery in load tests. This diff is just a fix to prevent system metrics from being published, as we don't need them in BigQuery